code/game/objects/items/weapons/tools.dm

```diff
@@ -843,6 +844,37 @@
 	reagents.add_reagent(SACID, 50)
 	update_icon()
 
+/obj/item/tool/solder/screw
+	name = "screwsolder"
+	desc = "An advanced solder with a screwdriver head. Use in hand to swap to and from the screwhead."
```

Comment:
"solder" refers to the alloy or metal used, the actual heating is done by a soldering iron or gun.